expyfun/_experiment_controller.py

```diff
@@ -933,6 +935,8 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
             A time relative to which timestamping is done. Ignored if
             timestamp==False.  If ``None``, timestamps are relative to the time
             `listen_presses` was last called.
+        releases : False | bool
+            Whether key-release events should be returned as well.
```

Comment:
I think the convention is to only include accepted *types* on the first line, not default values.  So it should read `releases : bool`.  Information about the default value goes in the following docstring line, so:

    Whether to include key-release events as well as key-press events (default ``False``).